processors/npzProcessor.py

```diff
@@ -0,0 +1,295 @@
+from coffea import hist, processor
+import numpy as np
+import awkward1 as ak
+import awkward
+from utils import utility as utl
+from utils.objects import Objects
+from utils import baseline as bl
+from itertools import combinations
+
+def col_accumulator(a):
+    return processor.column_accumulator(np.array(a))
+
+class MainProcessor(processor.ProcessorABC):
+        def __init__(self):
+                self._accumulator = processor.dict_accumulator({
+                "evtw": processor.column_accumulator(np.zeros(shape=(0))),
```

Comment:
we should try to pass the `variables` list (suggested above) into here, and use a similar dictionary comprehension to reduce duplication